### PR TITLE
feature/transfer-apps

### PR DESCRIPTION
**Pull request by elmerv**
This pr introduces theTransfer apps feature. This feature is located in the instance settings of every app and allows users to transfer an app to any organization they are member of.

### CODE REVIEW

File: source/components/builderv2/AppTransferManager.vue [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    this.loadOrganizations();
+  },
+
+  methods: {
+    ...mapMutations(["updateAppOwner"]),
+
+    isOwner(org_id, org_name) {
+      // console.log(this.current_owner_id, org_id, owner_type)
+      if(this.current_owner_id === org_id && this.owner_type === 'Organization'){
+        this.organization_name = org_name
```
Comment (vgkids) [inline source/components/builderv2/AppTransferManager.vue:37]: A method named `isOwner` should return a true/false answer, and not perform a change while it does the lookup.

If you make `organization_name` a computed property, you don't need to set it at all.

File: source/components/builderv2/AppTransferManager.vue [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+import { fetchJSON } from "trivial-core/lib/component-utils";
+import { mapState } from "vuex";
+import { mapMutations } from "vuex";
+
+export default {
+  computed: {
+    ...mapState({
+      descriptive_name: (state) => state.app.descriptive_name,
+      current_owner_id: (state) => state.app.owner_id,
+      owner_type: (state) => state.app.owner_type,
```
Comment (vgkids) [inline source/components/builderv2/AppTransferManager.vue:11]: Make the naming convention match. The matching pair of  `current_owner_id` would be `current_owner_type`.

**Review by vgkids — CHANGES_REQUESTED** [review]
Minor tweaks, looks good overall.

File: source/components/builderv2/AppTransferManager.vue (hunk 1 above)
Comment (elmerv) [reply source/components/builderv2/AppTransferManager.vue:37]: I just spent some time learning about computed properties, but I can't figure out how implementing a computed property for organization_name would be an alternative. The organization_name details the organization that owns the app and that information is only accessible through org.name inside v-for. Could you provide more clarification or an example on how computed property would be an alternative? I can also change the `isOwner()` function name to `checkAndSetOwner()` to check and set the organization_name.

Comment (elmerv) [reply source/components/builderv2/AppTransferManager.vue:37]: Sounds good. Thank you for the tip, I'll definitely look out for them and look into computed properties in my off time. I'll come back to this code, once I've figured it out. Thanks again for the help and clarification.

**Review by vgkids — APPROVED** [review]

**Pull request merged**